javascript/lib/Arrow_generated.d.ts

```diff
@@ -0,0 +1,5 @@
+export var org: {
+  apache: {
+    arrow: any
+  }
+}
```

Comment:
There's a WIP PR for adding TypeScript support to flatbuffers ([#4065](https://github.com/google/flatbuffers/pull/4065)), which should make this definition unnecessary.